core/arch/arm/plat-imx/conf.mk

```diff
@@ -502,23 +502,22 @@ CFG_IMX_OCOTP ?= y
 
 # Almost all platforms include CAAM HW Modules, except the
 # ones forced to be disabled
-CFG_NXP_CAAM ?= n
+CFG_NXP_CAAM ?= y
```

Comment:
This changes `CFG_NX_CAAM` default value for most boards. Actually this change rather reflects twhat is stated in the above inline comment. I think this change deserves a specific commit "plat-imx: default enable caam where supported". Unless I messed something...

Reply:
indeed, it was modified here https://github.com/OP-TEE/optee_os/commit/bfabce223d98a5e67f54cea6417b2481f7a64943
let's keep at it was and reword the inline comment